torchvision/ops/deform_conv.py

```diff
@@ -33,6 +34,9 @@ def deform_conv2d(
         padding (int or Tuple[int, int]): height/width of padding of zeroes around
             each image. Default: 0
         dilation (int or Tuple[int, int]): the spacing between kernel elements. Default: 1
+        mask (Tensor[batch_size, offset_groups * kernel_height * kernel_width,
+            out_height, out_width]): masks to be applied for each position in the
+            convolution kernel.
```

Comment:
For a follow-up PR, it might be good to mention that now `DeformConv2d` implements modulation from https://arxiv.org/abs/1811.11168 as well (also known as DeformConv v2)